package.json

```diff
@@ -84,8 +84,7 @@
     "word-wrap": "1.2.4",
     "electron-builder/app-builder-lib/@electron/universal/dir-compare/minimatch": "^3.0.5",
     "**/semver": "^7.5.2",
-    "rawproto/protobufjs": "^7.2.4",
-    "**/follow-redirects": "^1.15.4"
```

Comment:
@GnaneshKunal Are you sure that we must remove this line?
@zalenskiSofteq @romansergeenkosofteq Do you happy to know why we've added it? Is it safe to remove?

Reply:
Yeah. Safe for removal. `follow-redirects` now has a new version released, `1.15.6`, which must be upgraded, but the NPM resolution has blocked the upgrade. So, it's necessary to remove this.